sdk/keyvault/azure-keyvault-keys/src/main/java/com/azure/security/keyvault/keys/cryptography/CryptographyServiceClient.java

```diff
@@ -0,0 +1,159 @@
+package com.azure.security.keyvault.keys.cryptography;
+
+import com.azure.core.http.rest.Response;
+import com.azure.core.util.Context;
+import com.azure.core.util.logging.ClientLogger;
+import com.azure.security.keyvault.keys.models.Key;
+import reactor.core.publisher.Mono;
+
+import java.net.MalformedURLException;
+import java.net.URL;
+import java.security.MessageDigest;
+import java.security.NoSuchAlgorithmException;
+import java.util.Objects;
+
+class CryptographyServiceClient {
```

Comment:
Is this an internal client that is not visible to customers using SDK?

Reply:
yeah, its internal.